packages/snaps-controllers/src/test-utils/controller.ts

```diff
@@ -454,6 +454,13 @@ export const DEFAULT_ENCRYPTION_KEY_DERIVATION_OPTIONS = {
   },
 };
 
+export const LEGACY_ENCRYPTION_KEY_DERIVATION_OPTIONS = {
```

Comment:
Maybe move this to the Snap controller file (or a constants file) and use it as the fallback, to remove some duplication?

Reply:
Done!